src/sysinfo/nvapi_adapter.h

```diff
@@ -29,6 +29,7 @@ namespace dxvk {
         [[nodiscard]] std::string NvmlErrorString(nvmlReturn_t result) const;
         [[nodiscard]] nvmlReturn_t NvmlDeviceGetTemperature(nvmlTemperatureSensors_t sensorType, unsigned int* temp) const;
         [[nodiscard]] nvmlReturn_t NvmlDeviceGetUtilizationRates(nvmlUtilization_t* utilization) const;
+        [[nodiscard]] nvmlReturn_t NvmlDeviceGetVbiosVersion(char *version, unsigned int length) const;
```

Comment:
Please bind the * to left side, thus `char* version`. I know, the styles are messy in this project, though I would prefer c++ style for our own types. Same for the method implementation.

Reply:
Please do that also for the implementation (in the cpp file.)